### ISSUE TITLE
Fix failing v2-beta subsumer frequencies test

### ISSUE THREAD

**Issue by johnbradley** [phenoscape/rphenoscape#214]
The `term frequencies for post-comp subsumers of entities` test is failing when using the [v2-beta kb.phenoscape API](https://dev.phenoscape.org/api/v2-beta)
```
Failure (test-freqs.R:142:3): term frequencies for post-comp subsumers of entities
any(freqs > 0) is not TRUE
```
https://github.com/phenoscape/rphenoscape/blob/b668eb99c7fd29dc24a2ee6dd4c7f19a48926aca/tests/testthat/test-freqs.R#L135-L142
The test fails because all frequencies are 0.

**Comment by johnbradley** [comment]
One noticeable difference between v2-beta and prod is the post-comp URLs returned by `subsumer_matrix()`.
Example v2-beta URL:
```
http://purl.org/phenoscape/kb/patterns/part_of.yaml#170ba1b5100b484b56d9bb57d53b7da28bb1fbf4
```

Example production URL:
```
http://purl.org/phenoscape/expression?value=%3Chttp%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FBFO_0000050%3E+some+%3Chttp%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FUBERON_0008897%3E
```
 
These URLs are coming from the `/similarity/matrix` kb.phenoscape API response.


For the production URLs (http://purl.org...expression...) the `decodeIRI` functionality in `annotations_count` replaces the URL with a simpler one such as `http://purl.obolibrary.org/obo/UBERON_0008897` for the example above.
https://github.com/phenoscape/rphenoscape/blob/82e16d96e3ed34c6ddd59ee640ce58310d684c62/R/term-

This simpler URL is passed to `/taxon/annotations` and retrieves a total > 0.
weights.R#L130-L134

**Comment by johnbradley** [comment]
Here is the function that simplifies the URL:
https://github.com/phenoscape/rphenoscape/blob/82e16d96e3ed34c6ddd59ee640ce58310d684c62/R/term-weights.R#L112-L122

**Comment by johnbradley** [comment]
## Summary
Previously the post-comp IRIs returned from subsumer_matrix could be broken into parts. The entity IRI would be extracted and /taxon/annotations would return a count for the entity IRI. The v2-beta API returns post-comp IRIs cannot be broken into parts to extract the entity IRI.

### Example
Post-comp IRI returned by `subsumer_matrix()` before switching to the new v2-beta API: 
```
http://purl.org/phenoscape/expression?value=%3Chttp%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FBFO_0000050%3E+some+%3Chttp%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FUBERON_0008897%3E
```
The above IRI was broken into parts and the entity extracted resulting in:
```
http://purl.obolibrary.org/obo/UBERON_0008897
```
This IRI was then passed to the /taxon/annotations endpoint and we received a count back.
```
> annotations_count(iri="http://purl.obolibrary.org/obo/UBERON_0008897", termType="entity")
[1] 13405
```

**Comment by johnbradley** [comment]
Discussed with @hlapp. The plan is to remove the ability to use the __taxon_annotations__ corpus with `term-freqs()`.

**Comment by johnbradley** [comment]
Fixed by #226
